app/src/ActionModal/components/ModalDeposit.vue

```diff
@@ -142,8 +126,7 @@ export default {
         amount: {
           amount: this.amount,
           denom: this.denom,
-        },
-        numberOfSeconds: this.numberOfSeconds,
```

Comment:
we need this for the transaction. Now the `depositTx` is broken for Polkadot

Reply:
didn't know that